src/cpp/Agent.cpp

```diff
@@ -58,22 +232,22 @@ bool Agent::create_client(
     result = root.create_client(client_representation, agent_representation);
     if (dds::xrce::STATUS_OK != result.status())
```

Comment:
With the last talk, I think that this condition must be removed, (modify the next occurrences).

Reply:
Absolutely true!